napari/components/add_layers_mixin.py

```diff
@@ -777,6 +777,98 @@ def add_vectors(
         self.add_layer(layer)
         return layer
 
+    def add_tracks(
+        self,
+        data,
+        *,
+        properties=None,
+        graph=None,
+        tail_width=2,
+        tail_length=30,
+        n_dimensional=True,
+        name=None,
+        metadata=None,
+        scale=None,
+        translate=None,
+        opacity=1,
+        blending='additive',
+        visible=True,
+        colormap='turbo',
+        color_by='track_id',
+        colormaps_dict=None,
+    ) -> layers.Tracks:
+        """Add a tracks layer to the layers list.
+
+        Parameters
+        ----------
+        data : array (N, D)
+            Coordinates for N points in D dimensions. ID,T,(Z),Y,X
```

Comment:
```suggestion
        data : array (N, D+1)
            Coordinates for N points in D+1 dimensions. ID,T,(Z),Y,X. The first axis is the integer ID of the track. D is either 3 or 4 for planar or volumetric timeseries respectively.
```
Is this correct? What would happen if D was 5, i.e. `ID, T, C, Z, Y, X`? I'm right in thinking the first axis after ID must be `T` which has special handling in the shader right? This is fine, I just want a bit more info in the doc string.

Reply:
So there's no reason why ID, T, C, Z, Y, X wouldn't work. Only issue is that `_validate_data` asserts that the data must be 2 or 3 spatial dimensions + ID + T.